src/libraries/System.Private.CoreLib/src/System/AggregateException.cs

```diff
@@ -213,18 +213,19 @@ public override void GetObjectData(SerializationInfo info, StreamingContext cont
         }
 
         /// <summary>
-        /// Returns the <see cref="AggregateException"/> that is the root cause of this exception.
+        /// Returns the <see cref="Exception"/> that is the root cause of this exception.
+        /// This will either be the root exception, or the first <see cref="AggregateException"/>
```

Comment:
What does "the root exception" mean here?

Reply:
The exception described as the _root cause_ in the initial comment. I've no idea how to phrase this in a clearer way.